src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -669,6 +676,10 @@ class InferenceProfiler {
   /// Whether server-side inference server metrics should be collected.
   bool should_collect_metrics_{false};
 
+  /// User set threshold above which the PA overhead is too significant to
+  /// provide useable results.
+  const double overhead_pct_threshold_{0.0};
```

Comment:
I haven't looked super close at how all of this fits together, but what is this for? It is const, and hardcoded to 0? Does it get modified in the constructor or something?

Reply:
This just gives it a default value if the constructor doesn't so that it is not undefined. 
The constructor will give it whatever value is set as the default in the command_line_parser.h.